index.html

```diff
@@ -67,56 +67,60 @@ <h2 class="h1">What we’re working on</h2>
                 <p class="short-desc">
                     We’re redesigning and reorganizing, section by section,
                     so you can find the information you need quickly and easily, on any device.
-                    These charts tell you how much of each section we've published in the new format.
                 </p>
+                <section class="project-progress u-hidden">
+                    <p class="short-desc">
+                        These charts tell you how much of each section we've published in the new format.
+                    </p>
 
-                <div class="progress-charts">
-                    <div class="progress-charts_container" id="consumer-guides">
-                        <canvas class="progress-charts_chart">
-                            <div class="chart-bg">
-                                <div class="bar-chart"></div>
-                            </div>
-                        </canvas>
-                        <h3 class="chart-name">Consumer Guides</h3>
-                        <div class="chart-value u-visually-hidden">{{ pct.consumer_guides }}</div>
+                    <div class="progress-charts">
```

Comment:
Would it be worth expanding the `pct` dict to include the chart names, reduce this to one block, and loop through the dict to create them all dynamically?